README.md

```diff
@@ -2,7 +2,7 @@
 
 `harp.gl` is an _experimental and work in progress_ open-source 3D map rendering engine.
 
-### [Documentation](https://heremaps.github.io/harp.gl/doc) - [Examples](https://heremaps.github.io/harp.gl/examples)
```

Comment:
How to find the examples now?

Reply:
With https://github.com/heremaps/harp.gl/pull/455
After we make a release, we will have in gh-pages a new html that will show current master and past releases in order, so it only makes sense to link directly to master gh-pages